src/System.Management.Automation/engine/MshCommandRuntime.cs

```diff
@@ -249,7 +249,7 @@ public void WriteObject(object sendToPipeline, bool enumerateCollection)
 
 #if CORECLR
             // SecurityContext is not supported in CoreCLR
-            DoWriteObjects(sendToPipeline);
+            DoWriteEnumeratedObject(sendToPipeline);
```

Comment:
Please make new PR for cleanups.

Reply:
Yeah, I suppose that makes the most sense. I'll cut&paste this commit to another PR. 🙂